lineman/app/services/thread_query_service.coffee

```diff
@@ -52,13 +54,15 @@ angular.module('loomioApp').factory 'ThreadQueryService', (Records) ->
           view.applyFind(lastActivityAt: { $gt: moment().startOf('day').subtract(6, 'week').toDate() })
           view.applyWhere (thread) -> thread.isUnread()
 
-      if _.contains(filters, 'show_muted')
-        view.applyFind(volume: { $eq: 'mute' })
-      else
-        view.applyFind(volume: { $ne: 'mute' })
-
       _.each filters, (filter) ->
         switch filter
+          when 'show_muted'
+            #view.applyFind($or: {volume: { $eq: 'mute' }, groupId: {$in: CurrentUser.mutedGroupIds()}})
```

Comment:
@gdpelican tried to use an Or statement and failed.. not really sure if its supported. I'm going to ask LokiJS